### ISSUE TITLE
Stucks at [INFO] Waiting for RiotClient to update...

### ISSUE THREAD

**Issue by aryahalakove** [DeviatorZ/league-account-checker#41]
Hi, it stucks at [INFO] Waiting for RiotClient to update...
client is manually updated and no update here but checker stucks

**Comment by aryahalakove** [comment]
Also in old version, it loops: ERROR - xxxx session exception: Launch timed out.

**Comment by DeviatorZ** [comment]
I have made some experimental changes. Let me know if they resolve the issues for you. Now, the patching status will be completely ignored when running in a multi-threaded environment.

**Comment by aryahalakove** [comment]
Yes it's solved, thanks for your support. Also there's a little bug or something I didn't understand really but if I understand that button correctly, delete raw data must delete the exports folder, right ? But it doesn't clearing exports.
